README.md

```diff
@@ -1,11 +1,11 @@
 # Qiskit Terra
 [![License](https://img.shields.io/github/license/Qiskit/qiskit-terra.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)<!--- long-description-skip-begin -->[![Release](https://img.shields.io/github/release/Qiskit/qiskit-terra.svg?style=popout-square)](https://github.com/Qiskit/qiskit-terra/releases)[![Downloads](https://img.shields.io/pypi/dm/qiskit-terra.svg?style=popout-square)](https://pypi.org/project/qiskit-terra/)[![Coverage Status](https://coveralls.io/repos/github/Qiskit/qiskit-terra/badge.svg?branch=main)](https://coveralls.io/github/Qiskit/qiskit-terra?branch=main)[![Minimum rustc 1.64.0](https://img.shields.io/badge/rustc-1.64.0+-blue.svg)](https://rust-lang.github.io/rfcs/2495-min-rust-version.html)<!--- long-description-skip-end -->
 
-**Qiskit** is an open-source framework for working with noisy quantum computers at the level of pulses, circuits, and algorithms.
+**Qiskit**  is an open-source SDK for working with quantum computers at the level of extended quantum circuits, operators, and functions.
```

Comment:
* `extended quantum circuits`: I suspect this means `dynamic quantum circuits` (or just `dynamic circuits`). Otherwise, I'm unsure about its meaning. I quick search in the documentation did not help
 * `operators`: Maybe this can link to https://qiskit.org/documentation/stubs/qiskit.quantum_info.Operator.html ?
 * `functions`: Without context, this word is too generic. Suggestion: `quantum primitives` or just `primitives`? I prefer the first one as it is less generic.

Reply:
We defined the extend quantum circuit in the openqasm and yes I agree we have lost this (I am not sure what version removed it) and we need to bring it back. It does includes dynamics etc. The good news is it is still in the description of qiskit.

Operators is the default way for working with quantum observables. They are an operator and we can call it quantum operators is you need (but there is too many quantums)

Functions are just the primitives and I want to keep it general you can put a quantum function there if need be.

Reply:
See the definition of "extended quantum circuits" here: https://arxiv.org/abs/2104.14722 (second paragraph of the introduction).

Linking to the Operator docs in quantum_info would be fine, but the terms is unlikely to be confused with other meanings of "operator" in this context.

@jaygambetta's use of "functions" is perhaps novel. Can we just say "primitives" for now?

Reply:
Im good with primitives but I do feel we need to own the visions of quantum functions as we go forward.

Reply:
how do I edit it to primitives in the web version of GitHub. I see that I could just commit your other changes?

Reply:
done.